tests/test_investpy.py

```diff
@@ -445,9 +445,27 @@ def test_investpy_indices():
     retrieve_index_countries(test_mode=True)
 
 
+def test_investpy_currencies():
```

Comment:
Both `tests/` for errors and successes should be implemented in order to keep codecov percentage over 90%. So on this test file should be improved or kept simple, but notifying the owner that before releasing new version the tests need to be updated.